pkg/components/storage.go

```diff
@@ -11,6 +13,23 @@ import (
 	"github.com/openshift/microshift/pkg/config/lvmd"
 )
 
+// getCSIPluginConfig searches for a user-defined lvmd configuration file in predefined locations are returns the first
+// found, else a default-value lvmd config. Returns an error if the file exists and cannot be read, or exists and cannot
+// be deserialized (indicating a malformed config).  User's home directory is checked first.  If not found, global
+// MicroShift config directory is checked.
+func getCSIPluginConfig() (*lvmd.Lvmd, error) {
+	userCfg := filepath.Dir(config.DefaultUserConfigFile)
+	globalCfg := filepath.Dir(config.DefaultGlobalConfigFile)
+
+	if _, err := os.Stat(userCfg); errors.Is(err, os.ErrNotExist) {
+		return lvmd.NewLvmdConfigFromFile(filepath.Join(lvmd.LvmdConfigFileName))
+	}
+	if _, err := os.Stat(globalCfg); errors.Is(err, os.ErrNotExist) {
+		return lvmd.NewLvmdConfigFromFile(filepath.Join(globalCfg, lvmd.LvmdConfigFileName))
+	}
```

Comment:
```suggestion
	userCfg := filepath.Join(filepath.Dir(config.DefaultUserConfigFile), lvmd.LvmdConfigFileName)
	globalCfg := filepath.Join(filepath.Dir(config.DefaultGlobalConfigFile), lvmd.LvmdConfigFileName)

	if _, err := os.Stat(userCfg); !errors.Is(err, os.ErrNotExist) {
		return lvmd.NewLvmdConfigFromFile(userCfg)
	}
	if _, err := os.Stat(globalCfg); !errors.Is(err, os.ErrNotExist) {
		return lvmd.NewLvmdConfigFromFile(globalCfg)
	}
```

Reply:
Issue here was that code checked if config dir existed and then tried to load nonexistent file from that dir. Changed logic to stat the lvmd file instead and in case of existence use it to create the config